hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -415,14 +418,30 @@ public boolean commit(String commitTime,
         HoodieActiveTimeline activeTimeline = table.getActiveTimeline();
 
         List<Tuple2<String, HoodieWriteStat>> stats = writeStatuses
-            .mapToPair((PairFunction<WriteStatus, String, HoodieWriteStat>) writeStatus ->
-                new Tuple2<>(writeStatus.getPartitionPath(), writeStatus.getStat()))
-            .collect();
+                    .mapToPair((PairFunction<WriteStatus, String, HoodieWriteStat>) writeStatus ->
```

Comment:
Please make sure the file formatting/spacing does not change withut code changes.. e,g these three lines..